src/config/dynamic-theme-fixes.config

```diff
@@ -25962,6 +25962,14 @@ CSS
 
 ================================
 
+spaceship.com
+
+INVERT
+.lf-logo.lf-color-scheme--light
+.lp-background-image
+
+================================
```

Comment:
If INVERT will be still needed then IGNORE INLINE STYLE will be as the next section under it to add.

Reply:
It appears this has been solved as well. Changed accordingly and thanks!